js_modules/dagit/packages/core/src/assets/AssetPartitions.tsx

```diff
@@ -33,21 +34,29 @@ interface Props {
   opName?: string | null;
 }
 
+const DISPLAYED_STATES = [PartitionState.MISSING, PartitionState.SUCCESS];
+
 export const AssetPartitions: React.FC<Props> = ({
   assetKey,
-  assetPartitionNames,
+  assetPartitionDimensions,
   assetLastMaterializedAt,
   params,
   setParams,
   liveData,
 }) => {
   const [assetHealth] = usePartitionHealthData([assetKey], assetLastMaterializedAt);
-  const [ranges, setRanges] = usePartitionDimensionRanges(assetHealth, assetPartitionNames);
+  const [ranges, setRanges] = usePartitionDimensionRanges(
+    assetHealth,
+    assetPartitionDimensions,
+    true,
+  );
 
-  const [stateFilters, setStateFilters] = React.useState<PartitionState[]>([
-    PartitionState.MISSING,
-    PartitionState.SUCCESS,
-  ]);
+  const [stateFilters, setStateFilters] = useQueryPersistedState<PartitionState[]>({
+    defaults: {states: DISPLAYED_STATES.sort().join(',')},
```

Comment:
Doesn't look like it matters much on this array, but remember that `sort()` is mutative

Reply:
Mmm good catch this is not good